src/stories/components/search-input/search-input.tsx

```diff
@@ -2,13 +2,16 @@ import React, { CSSProperties, useState } from 'react';
 import styled from '@emotion/styled';
 import Magnifier from '../svg/magnifier';
 import { useThemeContext } from '../../../core/context/ThemeContext';
+import { Close } from '../svg/close';
 
 interface SearchInputProps {
   value?: string;
   defaultValue?: string;
   placeholder: string;
   onChange?: (text: string) => void;
   style?: CSSProperties;
+  handleCloseSearch?: () => void
+  inputRef?: React.LegacyRef<HTMLInputElement>
```

Comment:
Please, use `React.RefObject<HTMLInputElement>` instead of the `React.LegacyRef<HTMLInputElement>` since `LegacyRef` can be removed in future versions of React